packages/oc-template-jade-compiler/lib/compileView.js

```diff
@@ -17,7 +16,7 @@ module.exports = (options, callback) => {
   const publishFileName = options.publishFileName || 'template.js';
 
   if (!fs.existsSync(viewPath)) {
-    return callback(format(strings.errors.VIEW_NOT_FOUND, viewFileName));
+    return callback(strings.errors.VIEW_NOT_FOUND(viewFileName));
```

Comment:
As discussed, being those functions, could we perhaps camelCase then instead of uppercase  ?